<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialNo</class>
 <widget class="QMainWindow" name="SerialNo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialNo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="browseFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Serial</string>
    </property>
   </widget>
   <widget class="QListWidget" name="pathOpen">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseDir">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Searching Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startSearch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
